out_writeapi.go

```diff
@@ -581,13 +605,15 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 
 		if (currsize + len(buf)) >= config.maxChunkSize {
 			// Appending Rows
-			err := sendRequest(ms_ctx, binaryData, &config)
+			// TODO: Change for desired stream, defaulting to 0 for now
+			err := sendRequest(ms_ctx, binaryData, &config, 0)
```

Comment:
How will the stream index be computed in future? Can we put that logic into a function that simply returns 0 for now?

Reply:
Done.